src/enums/stat-names.enum.ts

```diff
@@ -0,0 +1,14 @@
+export enum StatNamesEnum {
+  healthPoints = 'Health Points',
+  AethrisPoints = 'Ætheris Points',
```

Comment:
This is not the same case the other words.
Also (like the constants by the way), enums are usually written in SNAKE_SCREAM_CASE

Reply:
Will do!